Conversion.java

```diff
@@ -0,0 +1,72 @@
+/*
+This class is used for conversion of one base to other base and to validate whether the given input is valid or not
+Owner -> Ojas Joshi
+Date -> 25/09/2024
+ */
+public class Conversion {
+    public static long othersToDecimal(String input, int base) {
+        int length = input.length() - 1;
+        long value = 0;
+        for (int i = 0; i < input.length(); i++) {
+            value = value + (Character.getNumericValue(input.charAt(length - i)) * Maths.pow(base, i));
+        }
+        return value;
+    }
+
+    public static boolean isValidHex(String input) {
+        for (int i = 0; i < input.length(); i++) {
+            if (input.charAt(i) != '1' && input.charAt(i) != '2' &&
+                    input.charAt(i) != '3' && input.charAt(i) != '4' &&
+                    input.charAt(i) != '5' && input.charAt(i) != '6' &&
+                    input.charAt(i) != '7' && input.charAt(i) != '8' &&
+                    input.charAt(i) != '9' && input.charAt(i) != '0' &&
+                    input.charAt(i) != 'A' && input.charAt(i) != 'B' &&
+                    input.charAt(i) != 'C' && input.charAt(i) != 'C' &&
+                    input.charAt(i) != 'D' && input.charAt(i) != 'E' &&
+                    input.charAt(i) != 'F' && input.charAt(i) != 'a' &&
+                    input.charAt(i) != 'b' && input.charAt(i) != 'c' &&

```

Comment:
You can optimize cases if you always convert lowercase to uppercase, or you can try regular expressions for checking all the validation of all bases. 